imagersite/photo/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from photo.models import Photos, Album
+from imagerprofile.models import ImagerProfile
+
+
+class ImagerForm(forms.ModelForm):
+    def __init__(self, user, *args, **kwargs):
+        self.photo_owner = user
+        super(forms.ModelForm, self).__init__(*args, **kwargs)
+
+    def save(self, *args, **kwargs):
```

Comment:
Again, it's a little odd to change the call semantics of a method like this.  Dangerous too if your form is used anywhere that you don't anticipate.  

It would be better to call the standard save with `commit=False` in the context of a view, and then do the work below in that context, not here.